datagateway_api/src/common/config.py

```diff
@@ -133,6 +133,9 @@ class SearchAPI(BaseModel):
     mechanism: StrictStr
     username: StrictStr
     password: StrictStr
+    scoring_enabled: StrictBool
+    scoring_server: StrictStr
+    scoring_group: StrictStr
```

Comment:
Could you add example values to `config.json.example` please? Keeping the scoring disabled might be best in case of someone using DataGateway API only, not the search API.

Reply:
This is my config:
```
    "scoring_enabled": true,
    "scoring_server": "http://dau-dm-01:9000/score?limit=2000",
    "scoring_group": "investigation"
```

Reply:
I guess this is your own scoring server? Does this mean I'll need to setup my own instance to test your changes? Is there a PaNOSC scoring server that I could use?

Reply:
No, there is not as far as I know and take into account that you need to install and populate with your own data in order to calculate the weights. 
It is not a big deal but has to be done.

Reply:
That's good to know, I will setup my own and test this branch when I get a chance